inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,27 @@ def create_record(record):
     return json
 
 
+def attach_documents_and_figures(record):
+    control_number = record.get('control_number', record.get('recid'))
+    for attachment in chain(record.get('documents', []), record.get('figures', [])):
+        file_url = attachment.get('url')
```

Comment:
same here, if you are expecting a key to exist in the dict you are using, instead of `get` use `[]` to fail quick if it's not there.